style/js/shield_defs.js

```diff
@@ -2158,6 +2158,57 @@ export function loadShields(shieldImages) {
     null
   );
 
+  // Sweden
+  shields["SE:RV"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white,
+    Color.shields.white,
+    4,
+    1,
+    null
```

Comment:
Do these shields have a fixed width or variable width? If the width is fixed, I would recommend this value be 30.

Reply:
They are variable in width, the more numbers in the ref, the wider the shield.